src/notation/internal/notationinteraction.cpp

```diff
@@ -1646,6 +1647,22 @@ bool NotationInteraction::applyPaletteElement(mu::engraving::EngravingItem* elem
 
     startEdit();
 
+    bool isMeasureAnchoredElement = (element->type() == ElementType::MARKER
```

Comment:
```suggestion
    const bool isMeasureAnchoredElement = element->type() == ElementType::MARKER
```
Ah, looks like we have an unwanted opening bracket here.

While you're fixing this - if you could also make it `const` and [squash your commits](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#squash) that would be perfect.

Many thanks!